src/ContaoCommunityAlliance/DcGeneral/EventListener/ModelRelationship/TreeEnforcingListener.php

```diff
@@ -77,15 +78,10 @@ public function process(EnforceModelRelationshipEvent $event)
      */
     private function handleInto(InputProviderInterface $input, ControllerInterface $controller, ModelInterface $model)
     {
-        $into = ModelId::fromSerialized($input->getParameter('into'));
+        $into = ModelId::fromSerialized($input->getParameter('pid'));
```

Comment:
This looks cumbersome. The correct parameter is "into", as the method name dictates.
Furthermore, who is now handling the "into" condition?